### ISSUE TITLE
Validation error messages not fully translated in response (I18n)

### ISSUE THREAD

**Issue by ruslankhaertdinov** [tiagopog/jsonapi-utils#56]
I have object with validation error. It correctly translated with I18n:
```
angle.errors.to_a
=> ["Камера не имеет сохранённых координат"] 
```
Translated as `Camera doesn't have coordindates` in `:en` locale.

But in request I got partly translated  response:
```
{"errors"=>[{"title"=>"Camera не имеет сохранённых координат", "id"=>"camera", "code"=>"100", "source"=>{"pointer"=>"/data/relationships/camera"}, "status"=>"422"}]}
```
Where validation message is not fully translated: `Camera` word not translated, but rest of message translated.

**Comment by tiagopog** [comment]
Nice catch, @ruslankhaertdinov!

Can you also provide a spec to the context "when validation fails with a locale other than :en" after the spec for the [context 'when validation fails with a formatted attribute key'](https://github.com/tiagopog/jsonapi-utils/blob/master/spec/controllers/posts_controller_spec.rb#L300)?

Thanks for contributing :-)

**Comment by ruslankhaertdinov** [comment]
Sure

**Comment by tiagopog** [comment]
I just re-commented at the PR, I think it's more appropriated there 👍

**Comment by tiagopog** [comment]
Closed by #57
